boards/xtensa/nxp_adsp_rt685/Kconfig

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) 2023 Google LLC.
+# Copyright (c) 2023 NXP
```

Comment:
I think it is better to add to copyright when the files are first introduced, or first modified in the PR.